### DISCUSSION TITLE
Scheduler validation

### DISCUSSION THREAD

**Discussion by collse** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#712] [Q&A]
I am trying to put a schedule in place but I am not sure if I understood the example correctly.

I want the hot tub to run on the weekends only

- FRI(PM)/SAT/SUN
- Pump running 24/7 (I prefer it this way)
- Heater daily push if necessary just after midnight
- Pump enabled just in case someone manually switched it off
- Heat set to 38 on Friday AM to be ready for PM
- Heat set to 30 degrees on SUN evening

this is my schedule but I wondered if this is correct?

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/5358625/bcfde0b9-f4d0-4e80-bd20-235c0815c73e)

**Comment by visualapproach** [comment]
Looks alright to me

**Reply by collse** [reply]
thanks mate - looked a bit "empty" hence the question

**Reply by visualapproach** [reply]
NP. I even think you can remove the 'pump on' because it should start with the heater.
